web-app/django/VIM/apps/instruments/models/instrument.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+
+class Instrument(models.Model):
+    wikidata_id = models.CharField(max_length=20, unique=True)
+    default_image = models.ForeignKey(
+        "AVResource",
+        on_delete=models.PROTECT,
+        blank=False,
+        null=False,
+        related_name="default_image_of",
+    )
+    hornbostel_sachs_class = models.CharField(max_length=50, blank=True)
+    mimo_class = models.CharField(max_length=50, blank=True)
```

Comment:
Guessing `mimo_class` is short for `mimo_classification`? This took me a moment. Perhaps either spell these out in full in the variable names, or add `help_text` for them.